conanfile.py

```diff
@@ -71,12 +65,21 @@ def build(self):
             source_folder=self.source_folder.replace("\\", "/"),
             install_folder=self.install_folder.replace("\\", "/")))
         cmake = CMake(self)
-        cmake.definitions["SEASOCKS_SHARED"] = self.options.shared
-        cmake.definitions["SEASOCKS_STATIC"] = self.options.static
         cmake.definitions["DEFLATE_SUPPORT"] = self.options.with_zlib
         cmake.configure(source_folder=self.build_folder)
         cmake.build()
 
     def package(self):
         cmake = CMake(self)
         cmake.install()
+
+    def package_info(self):
+        # Set the name of the generated `FindSeasocks.cmake` and `SeasocksConfig.cmake` cmake scripts
+        self.cpp_info.names["cmake_find_package"] = "Seasocks"
+        self.cpp_info.names["cmake_find_package_multi"] = "Seasocks"
+        self.cpp_info.components["libseasocks"].libs = ["seasocks"]
```

Comment:
```suggestion
        self.cpp_info.components["libseasocks"].libs = ["seasocks"]
        if self.settings.os == "Linux":
            self.cpp_info.components["libseasocks"].system_libs = ["pthread"]
```

Reply:
I made these changes and now:

```
[100%] Linking CXX executable bin/seasocks_test
/usr/bin/ld: CMakeFiles/seasocks_test.dir/seasocks_test.cpp.o: undefined reference to symbol 'pthread_create@@GLIBC_2.2.5'
/usr/bin/ld: /lib/x86_64-linux-gnu/libpthread.so.0: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
make[2]: *** [CMakeFiles/seasocks_test.dir/build.make:85: bin/seasocks_test] Error 1
make[1]: *** [CMakeFiles/Makefile2:76: CMakeFiles/seasocks_test.dir/all] Error 2
make: *** [Makefile:84: all] Error 2
ERROR: seasocks/1.4.4 (test package): Error in build() method, line 12
	cmake.build()
	ConanException: Error 2 while executing cmake --build '/home/mgodbolt/dev/personal/seasocks/test_package/build/7e3f3c9f8b0319ca5c01315759aeda2f877da74f' '--' '-j36'
```

Reply:
@mattgodbolt A static seasocks needs pthread on Linux

Reply:
Is pthread exposes through your headers or do you use pthread explicitly in the test packge?